src/main/java/com/cloudbees/jenkins/support/util/WordReplacer.java

```diff
@@ -0,0 +1,168 @@
+package com.cloudbees.jenkins.support.util;
```

Comment:
The new custom class to replace by words without loosing performance. There is a set of public methods for comfort.